tests/spec/core/issues-notes-spec.js

```diff
@@ -1,5 +1,12 @@
 "use strict";
 describe("Core — Issues and Notes", () => {
+  let utils;
+  beforeAll(done => {
```

Comment:
I don't think you need this import. Instead, for example, you can just do the following below without `parents()`:

```JS
expect(spiss.closest("div")).toBe(null);
```

Also, "toEqual()" means slightly different thing to `.toBe()`. In general, use `.toBe()` because it means "===". Some nice details here:
https://stackoverflow.com/questions/22413009/jasmine-javascript-testing-tobe-vs-toequal

Reply:
Gotcha. Thanks!